platform/ios/src/MGLMapView.mm

```diff
@@ -3759,6 +3785,29 @@ - (MGLAnnotationTag)annotationTagAtPoint:(CGPoint)point persistingResults:(BOOL)
     });
 }
 
+- (std::vector<MGLAnnotationTag>)shapeAnnotationTagsInRect:(CGRect)rect
+{
+    mbgl::ScreenBox screenBox = {
+        { CGRectGetMinX(rect), CGRectGetMinY(rect) },
+        { CGRectGetMaxX(rect), CGRectGetMaxY(rect) },
+    };
+    
+    std::vector<MGLAnnotationTag> nearbyAnnotations;
```

Comment:
This appears to be a list of all the annotation tags, not just the nearby ones.

Reply:
It contains nearby ones. You can look at line 3799:
```
return nearbyAnnotations = _mbglMap->queryShapeAnnotations(screenBox, { optionalLayerIDs });
```
Otherwise returns an empty vector.

Reply:
What I meant was that this vector contains all the annotations within the specified rect. It just happens that this method is called with a rect that represents the area near the tap gesture, but that isn’t required by this method. Perhaps `annotationsInRect` would be more self-explanatory.